From 96a0566a2c3d6faed31a5e275d9ba972baaaa56b Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Fri, 3 Feb 2017 10:48:17 -0600
Subject: [PATCH 087/205] Documentation: dt: Update PRU bindings with label
 property

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

Add a mandatory 'label' property to the PRU bindings. This property
will be used to exactly identify the PRU instance within a PRU-ICSS
remote processor subsystem.

Signed-off-by: Suman Anna <s-anna@ti.com>
(cherry picked from commit 01e1214e9eac286993b259fe0017729b50a6ed56)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../bindings/remoteproc/ti,pruss-remoteproc.txt    |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/remoteproc/ti,pruss-remoteproc.txt b/Documentation/devicetree/bindings/remoteproc/ti,pruss-remoteproc.txt
index 65991fb..18f49a5 100644
--- a/Documentation/devicetree/bindings/remoteproc/ti,pruss-remoteproc.txt
+++ b/Documentation/devicetree/bindings/remoteproc/ti,pruss-remoteproc.txt
@@ -163,6 +163,8 @@ Required Properties:
                        "iram" for Instruction RAM,
                        "control" for the CTRL sub-module registers,
                        "debug" for the Debug sub-module registers,
+- label          : should be either "pru0" or "pru1" identifying the PRU core
+                   within the PRUSS
 
 Optional Properties:
 --------------------
-- 
1.7.9.5

